ui/DesignSystem/Component/NotificationFaucet.svelte

```diff
@@ -9,7 +9,7 @@
   import { blur } from "svelte/transition";
 
   export let style = null;
-  let messages = [];
+  let messages = null;
```

Comment:
This actually is not a prop and initialising it with `null` throws an exception on app start.

<img width="443" alt="Screenshot 2020-03-02 at 15 31 23" src="https://user-images.githubusercontent.com/158411/75685498-ef6f7700-5c9a-11ea-8b5c-d18569959389.png">